src/app/Library/CrudPanel/Traits/SaveActions.php

```diff
@@ -123,32 +332,94 @@ public function performSaveAction($itemId = null)
                 'success'      => true,
                 'data'         => $this->entry,
                 'redirect_url' => $redirectUrl,
+                'referrer_url' => $referrer_url,
             ];
         }
 
+        if (isset($referrer_url)) {
+            session()->flash('referrer_url_override', $referrer_url);
+        }
+
         return \Redirect::to($redirectUrl);
     }
 
     /**
-     * Get the translated text for the Save button.
+     * Returns the default Backpack save actions for cruds.
      *
-     * @param string $actionValue
+     * @return array
+     */
+    public function setupBackpackDefaultSaveActions()
+    {
+        $defaultSaveActions = [
```

Comment:
Can I just say... I love how you've set up SaveActions, how clear they are now. So similar to fields / columns / filters / etc. Great job!

Reply:
You keep raising the bar! You'r the man \o